### DISCUSSION TITLE
I just wanted to make a blog

### DISCUSSION THREAD

**Discussion by Notjack113** [community/community#80609] [Other Feature Feedback, Questions, & Ideas]
I just wanted a website to post articles about gaming but instead I have to create projects about coding? any time i start a new project it's excel style spreadsheets. I'm sorry bu this isnt why I was told to come here and I cant figure out how to just create a blog style page.

**Comment by varunpats** [comment]
Hi, could you please consider watching YT videos about creating blog style pages? Because it is difficult to tell by just commenting, it's much better if you select your tech stack and search them for video tutorials. I'm sure you'll get multiple choices from where you can select one.
